plugins/discourse-narrative-bot/spec/system/user_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+describe "Narrative Bot PM", type: :system do
+  fab!(:admin)
+  fab!(:user)
+  fab!(:topics) { Fabricate.times(2, :post).map(&:topic) }
+  fab!(:posts) { Fabricate.times(3, :post, topic: topics[0]) }
+
+  context "when user tips are enabled" do
+    before do
+      Jobs.run_immediately!
+      SiteSetting.enable_user_tips = true
+      SiteSetting.discourse_narrative_bot_enabled = true
+      # shortcut to generate welcome post since we're not going through user creation or first login
+      user.enqueue_bot_welcome_post
+    end
+
+    it "does not delete the narrative bot PM when skipping all tips" do
+      sign_in user
+      visit "/"
+
+      find(".fk-d-tooltip__content .user-tip__buttons .btn", text: "Skip tips").click
```

Comment:
I should write a page object for this :)

Reply:
I did it: https://github.com/discourse/discourse/commit/2fdee7849cc6c79747f6cd2fa607bf6372ee1dd9

Reply:
Thanks!